lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -160,9 +160,11 @@ def self.fetch_existing_oc_binary_path_in_windows
         path_string = ENV['PATH']
 
         separator = ':' unless path_string.include?(';')
-        path_string.split(separator).detect do |dir_path|
+        oc_path_dir = path_string.split(separator).detect do |dir_path|
           File.exist? "#{dir_path}\\oc.exe"
         end
+
+        oc_path_dir.nil? ? nil : oc_path_dir + '\oc.exe'
```

Comment:
@hferentschik I had already explained it :smile: 

So, in windows, devsuite download the oc binary and place it in `C:\DevSuite\cdk\bin\oc.exe` which is also added in environment variable $PATH.

Previously, I was only checking through the directory paths presents in $PATH and checking the existence of dir_path + '\oc.exe' file. On first detection, I return the dir_path only. However, I need the actual full path not `dir_path`. 
Now, sometime if  detect doesn't match anything then it returns nil. 

Probably, I should have changed this line to `oc_path_dir + '\oc.exe' unless oc_path_dir.nil?`